src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -16,9 +16,20 @@
       Overwrite="true"
       Encoding="ASCII" />
 
+    <GetDependencyInfo 
+      DotnetInstallerCommit="$(BUILD_SOURCEVERSION)"
+      DependencyName="Microsoft.NETCore.App.Ref">
+       <Output TaskParameter="DependencyVersion" PropertyName="DepRuntimeVersion" />
+       <Output TaskParameter="DependencyCommit" PropertyName="DepRuntimeCommit" />
+    </GetDependencyInfo>
+
+    <!-- Format for productCommit-%rid%.txt:
+       installer:%commit%, %version
+       runtime:%commit%, $version
+    -->
     <WriteLinesToFile
       File="$(ArtifactsShippingPackagesDir)productCommit-$(Rid).txt"
-      Lines="$(BUILD_SOURCEVERSION)%0A$(PackageVersion)"
+      Lines="installer%3A$(BUILD_SOURCEVERSION)%2C%20$(PackageVersion)%0Aruntime%3A$(DepRuntimeCommit)%2C%20$(DepRuntimeVersion)%0A"
```

Comment:
I think there are various things that consume this file, and we can't update the format without breaking them.  It might be easier to create a new file for the runtime version.

Reply:
Do we know what those things are? Making a new file would be fine, but it is going to take a lot more work to plumb it through to the README.

It we want programs to consume this information, maybe we should make a new file that programs can consume, and that we can add to over time without breaking those programs.

Reply:
Changed PR to produce a new file `productCommits-$(Rid).txt`

Reply:
changed to `productCommit-$(Rid).txt` 🙂